server/react-admin/src/form/CheckboxInput.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Controller } from 'react-hook-form'
+import Checkbox from '@mui/material/Checkbox'
+import FormControlLabel from '@mui/material/FormControlLabel'
+
+CheckboxInput.propTypes = {
+  label: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  control: PropTypes.object.isRequired,
+  defaultValue: PropTypes.string,
+  required: PropTypes.bool,
+}
+
+function CheckboxInput({ label, name, control, defaultValue, required }) {
```

Comment:
in order to display errors we need to grab the errors off of the control object.

```
  const { errors } = useFormState({ control })
  const errorMessage = errors[name] ? errors[name].message : ''
```